homeassistant/components/tesla/climate.py

```diff
@@ -3,7 +3,7 @@
 
 from homeassistant.components.climate import ClimateDevice
 from homeassistant.components.climate.const import (
-    HVAC_MODE_HEAT,
+    HVAC_MODE_AUTO,
```

Comment:
I am surprised that this fixes the issue.

If it didn't work with `HVAC_MODE_HEAT` (and thus show a heat symbol), why would it now start showing an auto symbol ?